modules/search/pages/search-request-params.adoc

```diff
@@ -112,29 +119,37 @@ To turn on document relevancy scoring in search results, remove the `score` prop
 
 |search_after |Array |No a|
 
-NOTE: If you use `search_after` in a search request, you can't use `search_before`. Both properties are included in the example code to show the correct syntax.
+NOTE: If you use `search_after` in a search request, you cannot use `search_before`. Both properties are included in the example code to show the correct syntax.
 
 Use `search_after` with `from/offset` and `sort` to control pagination in search results. 
 
-Give a value for each string or JSON object in the `sort` array to the `search_after` array.
+For example, if you had a set of 10 documents to sort based on `_id` values of 1-10, with `from` set to `2` and `search_after` set to `8`, documents 9-10 appear on the same page. 
```

Comment:
I can't make head or tail of this, but I am a simple squirrel.

Reply:
Oh, though it's not new wording, just been moved... (still don't understand it though)